plugins/ingest-attachment/build.gradle

```diff
@@ -2106,7 +2106,27 @@ thirdPartyAudit.excludes = [
   'ucar.nc2.dataset.NetcdfDataset'
 ]
 
-if (JavaVersion.current() > JavaVersion.VERSION_1_8) {
+if (project.runtimeJavaVersion == JavaVersion.VERSION_1_8) {
+    thirdPartyAudit.excludes += [
+            // TODO: Why is this needed ?
+            'com.sun.javadoc.ClassDoc',
```

Comment:
I'm a bit puzzled why this came up now. 
AFAIK this is part of the jdk, and not the jre ( javac ). Maybe the ant runner did something to load that.